src/util/output.c

```diff
@@ -9,6 +9,8 @@
 #include "global.h"
 #include "util/output.h"
 
+BOOL output_video = FALSE; // NOTE: Must be default to FALSE until config.txt is loaded and after video initialization is done.
```

Comment:
```suggestion
BOOL output_video = FALSE; // NOTE: Must be set to a default value of FALSE until config.txt is loaded and after video initialization is done.
```

Reply:
fixed, also corrected "after video initialization is done" to "before video initialization".